libyul/optimiser/FunctionCallFinder.cpp

```diff
@@ -21,18 +21,39 @@
 using namespace solidity;
 using namespace solidity::yul;
 
-std::vector<FunctionCall*> FunctionCallFinder::run(Block& _block, YulString _functionName)
+namespace
+{
+template<typename Base, typename ResultType>
+class FunctionCallFinderBase: Base
```

Comment:
It's no longer used as a base class, is it?
```suggestion
class FunctionCallFinderImpl: Base
```

But this would probably better describe its purpose:
```suggestion
class MaybeConstFunctionCallFinder: Base
```